mne/commands/mne_show_fiff.py

```diff
@@ -21,11 +21,6 @@ def run():
     if len(args) != 1:
         parser.print_help()
         sys.exit(1)
-    # This works around an annoying bug on Windows for show_fiff, see:
-    # https://pythonhosted.org/kitchen/unicode-frustrations.html
-    if int(sys.version[0]) < 3:
-        UTF8Writer = codecs.getwriter('utf8')
-        sys.stdout = UTF8Writer(sys.stdout)
```

Comment:
this was certainly added for a reason :( I am not confident removing it like this...